string room_name
geometry_msgs/Pose pose
---
bool success
string msg